CppCoreGuidelines.md

```diff
@@ -12159,7 +12159,7 @@ When unambiguous, the `T` can be left out of `T{e}`.
 
     complex<double> f(complex<double>);
 
-    auto z = f({2*pi, 1});
+    auto z = f({2 * pi, 1});
```

Comment:
thanks for the pointer. Not sure if that discussion would have an impact on this location as well. I can remove from this PR and create a separate one if that helps.